doc/classes/@GlobalScope.xml

```diff
@@ -2316,41 +2316,41 @@
 		<constant name="KEY_Z" value="90" enum="Key">
 			Z key.
 		</constant>
-		<constant name="KEY_BRACKETLEFT" value="91" enum="Key">
-			[ key.
+		<constant name="KEY_BRACKETLEFT" value="91" enum="Key" keywords="KEY_SQUARE_BRACKET_LEFT">
+			Left bracket ([code][[/code]) key.
 		</constant>
 		<constant name="KEY_BACKSLASH" value="92" enum="Key">
-			\ key.
+			Backslash ([code]\[/code]) key.
 		</constant>
-		<constant name="KEY_BRACKETRIGHT" value="93" enum="Key">
-			] key.
+		<constant name="KEY_BRACKETRIGHT" value="93" enum="Key" keywords="KEY_SQUARE_BRACKET_RIGHT">
+			Right bracket ([code]][/code]) key.
```

Comment:
```suggestion
			Right bracket ([code skip-lint]][/code]) key.
```

Reply:
Somehow I completely forgot about this. Thank you.